### PR TITLE
keg-components - custom loading component

### PR DESCRIPTION
**Pull request by daniel112**
[Semver Status] minor

**Ticket**: [issue_id](https://jira.simpleviewtools.com/browse/ZEN-363)

## Context

* For the evf processing button. we want to use a custom icon instead of the default `react-native` one
* the keg-components Loading component allows for a custom Icon, but we need to have control over the animation

## Goal

* Update `useFromToAnimation` to allow looping 
* update the `Loading` story to include an example of a custom loading icon

## Updates

- `repos/keg-components/src/assets/icons/faLoading/faLoading.js`
    - added font-awesome spinner svg component for testing custom loading icon
- `repos/keg-components/src/components/loading/loading.js`
    - updated jsdoc
- `repos/keg-components/src/components/loading/loading.stories.js`
   - updated loading story for custom icon
- `repos/keg-components/src/hooks/useFromToAnimation.js`
   - updated `fromToAnimation` to allow looping

## Testing

* Run `keg dpg run --package docker.pkg.github.com/simpleviewinc/keg-packages/keg-components:custom-loading`
* navigate to http://local.kegdev.xyz/
* Checkout the Loading > Custom story item

### CODE REVIEW

**Review by lancetipton — COMMENTED** [review]
Looks good, just one small update and should be good

**Review by lancetipton — CHANGES_REQUESTED** [review]
Forgot to select Requesting changes

File: repos/keg-components/src/hooks/useFromToAnimation.js [hunk 1]
```diff
@@ -9,14 +9,17 @@ import { isArr } from '@keg-hub/jsutils'
...
+ * @param {Easing} params.easing - react-native Easing obj
  * @param {Function} params.onFinish - passed to Animated.start(), called when the animation finishes
  * @param {Array} dependencies - optional array of dependencies that will dictate when the hook re-runs the animation.
  *                               By default, the animation reruns if the `from` or `to` values change.
  * @returns {Array} [ Animated.Value ]
  *  - Animated.Value: pass this value to your Animated.View to begin the animation
  */
 export const useFromToAnimation = (params, dependencies) => {
-  const { from, to, duration = 500, onFinish = noOp } = params || {}
+  const { from, to, duration = 500, onFinish = noOp, loop = false, easing } =
```
[the remark this answers is not in the document]
Comment (daniel112) [reply repos/keg-components/src/hooks/useFromToAnimation.js:18]: updated.

also did a bit of digging on the behavior of unmounting an Animated content. 
https://stackoverflow.com/questions/59414569/should-i-explicitly-stop-react-native-animations-on-unmount#:~:text=The%20short%20answer%20is%20no,is%20bound%20to%20the%20Animated.

[the remark this answers is not in the document]
Comment (lancetipton) [reply repos/keg-components/src/hooks/useFromToAnimation.js:18]: Cool, thanks for looking into to it. Looks like it's not needed, and now we know.

**Review by lancetipton — APPROVED** [review]
Looks good. Thanks man

**Pull request merged**